node-sso-example/views/login_successful.ejs

```diff
@@ -4,36 +4,56 @@
     </head>
     <body class="container_success">
         <div class="logged_in_nav">
-            <div class="flex">                
-                <p>You're logged in <%=first_name%>, welcome!</p>
+            <div class="flex">
+                <div>
+                    <img src="../public/images/workos_logo_new.png" alt="workos logo">
```

Comment:
Could you swap this logo for the transparent one with the text included? 
![CleanShot 2022-08-02 at 14 23 26](https://user-images.githubusercontent.com/47123183/182466567-4869b7dc-f1c6-43f1-8a27-26e9af91ac63.png)